accord-core/src/test/java/accord/burn/TopologyUpdates.java

```diff
@@ -154,7 +153,8 @@ public static <T> BiConsumer<T, Throwable> dieOnException()
         return (result, throwable) -> {
             if (throwable != null)
             {
-                logger.error("", throwable);
+                logger.error("Unexpected exception", throwable);
+                logger.error("", new Throwable("Shutting down test"));
```

Comment:
hit this once and was useful to know where shutdown was happening and not just what triggered it